packages/fetchai/connections/ledger/ledger_dispatcher.py

```diff
@@ -145,16 +145,23 @@ def get_state(
         api: LedgerApi,
         message: LedgerApiMessage,
         dialogue: LedgerApiDialogue,
+        raise_on_try: bool = False,
```

Comment:
Please revert. This cannot be configured by the skill (only the message). Basically the implementation here has to determine whether to raise or not. If on line 162 you set it to True then when this gets processed it will return an error message response to the skill in case an exception is triggered on the try.

Reply:
so do I just need to revert it and remove it from the get_state method altogether on the ledger methods as well? Since we cannot configure this on the skill, so I need to make this a configurable on the message (`ledger_api/message.py::LedgerApiMessage`)

Reply:
See below